doc/nrf/app_dev/device_guides/nrf70/nrf7002eb2_dev_guide.rst

```diff
@@ -0,0 +1,128 @@
+      | IOVDDEN                           | P1.04             | Enable IOVDD regulator                        |
+      +-----------------------------------+-------------------+-----------------------------------------------+
+      | IRQ (IRQ)                         | P1.10             | Interrupt from nRF7002                        |
+      +-----------------------------------+-------------------+-----------------------------------------------+
+      | GRANT (GRT)                       | P1.03             | Coexistence grant from nRF7002                |
+      +-----------------------------------+-------------------+-----------------------------------------------+
+      | REQ (REQ)                         | P1.09             | Coexistence request to nRF7002                |
+      +-----------------------------------+-------------------+-----------------------------------------------+
+      | STATUS (ST0)                      | P1.08             | Coexistence status from nRF7002               |
+      +-----------------------------------+-------------------+-----------------------------------------------+
+
+DIP switch settings for nRF54L15 DK and nRF54H20 DK
+***************************************************
+
+The **GPIO MAP SEL** switch on the nRF7002-EB II can be used to select the GPIO mapping for the nRF54H20 DK and the nRF54L15 DK.
+
+* **nRF54L15 DK**: Set the switch towards MAP1 (the LED will glow at the **MAP1** end).
+* **nRF54H20 DK**: Set the switch towards MAP2 (the LED will glow at the **MAP2** end).
+
+.. _nrf7002eb2_building_programming:
+
+Building and programming with nRF7002-EB II
+*******************************************
+
+To build for the nRF7002-EB II, use a compatible :ref:`board target <app_boards_names>` with the CMake ``SHIELD`` option set to the corresponding shield name.
+See :ref:`cmake_options` for instructions on how to provide CMake options.
+
+For example, if you build the :ref:`wifi_shell_sample` sample for nRF54L15 DK on the command line, use the following command:
+
+.. code-block:: console
+
+   west build -p -b nrf54l15dk/nrf54l15/cpuapp -- -Dshell_SHIELD="nrf7002eb2" -Dshell_SNIPPET=nrf70-wifi
+
+If you use the |nRFVSC|, specify ``-DSHIELD=nrf7002eb2`` in the **Extra Cmake arguments** field when `setting up a build configuration <How to work with build configurations_>`_.
+
+Alternatively, add the shield in the project's :file:`CMakeLists.txt` file by using the following command:
+
+.. code-block:: console
+
+   set(SHIELD nrf7002eb2)
+
+To build for the nRF7002-EB II with nRF54H20 DK, use the ``nrf54h20dk/nrf54h20/cpuapp`` board target with the CMake ``SHIELD`` variable set to ``nrf7002eb2``.
+To build for a custom target, set ``-DSHIELD="nrf7002eb2"`` when you invoke ``west build`` or ``cmake`` in your |NCS| application.
+Alternatively, you can add the shield in the project's :file:`CMakeLists.txt` file by using the ``set(SHIELD nrf7002eb2)`` command.
```

Comment:
```suggestion
* To build for the nRF7002-EB II with nRF54H20 DK, use the ``nrf54h20dk/nrf54h20/cpuapp`` board target with the CMake ``SHIELD`` variable set to ``nrf7002eb2``.
* To build for a custom target, set ``-DSHIELD="nrf7002eb2"`` when you invoke ``west build`` or ``cmake`` in your |NCS| application.

Alternatively, you can add the shield in the project's :file:`CMakeLists.txt` file by using the ``set(SHIELD nrf7002eb2)`` command.
```

Reply:
I suggest to get this PR merged. I will take care of this under doc improvement phase.

Reply:
It's done.